tests/test_brownian_interval.py

```diff
@@ -0,0 +1,311 @@
+                                       levy_area_approximation=levy_area_approximation)
+
+        for _ in range(REPS):
+            t_ = npr.uniform(low=t0_, high=t1_)
+            ta = npr.uniform(low=t0_, high=t1_)
+            tb = npr.uniform(low=t0_, high=t1_)
+            ta, t_, tb = sorted([t_, ta, tb])
+
+            increment = bm(ta, tb).cpu().detach().numpy()
+            sample = bm(ta, t_).cpu().detach().numpy()
+
+            mean = increment * (t_ - ta) / (tb - ta)
+            std = np.sqrt((tb - t_) * (t_ - ta) / (tb - ta))
+            rescaled_sample = (sample - mean) / std
+
+            _, pval = kstest(rescaled_sample, 'norm')
+            assert pval >= ALPHA
+
+
+@pytest.mark.parametrize("device", devices)
+@pytest.mark.parametrize("levy_area_approximation", ['none', 'space-time', 'davie', 'foster'])
+@pytest.mark.parametrize("random_order", [False, True])
+def test_continuity(device, levy_area_approximation, random_order):
+    if device == gpu and not torch.cuda.is_available():
+        pytest.skip(msg="CUDA not available.")
+
+    ts = torch.linspace(0., 1., 10000, device=device)
+    bm = torchsde.BrownianInterval(t0=ts[0], t1=ts[-1], shape=(), device=device,
+                                   levy_area_approximation=levy_area_approximation)
+    vals = torch.empty_like(ts)
+    i_ = torch.arange(len(ts), device=device)
+    if random_order:
+        i_ = i_[torch.randperm(len(ts), device=device)]
+    for i in i_:
+        t = ts[i]
+        vals[i] = bm(t)
+    last_val = vals[0]
+    for val in vals[1:]:
+        assert (val - last_val).abs().max() < 5e-2
+        last_val = val
+
+
+@pytest.mark.skip("BrownianInterval.to not supported")
+@pytest.mark.parametrize("device", devices)
+@pytest.mark.parametrize("levy_area_approximation", ['none', 'space-time', 'davie', 'foster'])
+def test_to_dtype(device, levy_area_approximation):
+    if device == gpu and not torch.cuda.is_available():
+        pytest.skip(msg="CUDA not available.")
+
+    ta, tb, bm = _setup(device, levy_area_approximation, (SMALL_BATCH_SIZE, D))
+    tc = torch.rand_like(ta)
+    td = torch.rand_like(ta)
+    tc, td = min(tc, td), max(tc, td)
+    te = torch.rand_like(ta)
+    tf = torch.rand_like(ta)
+    te, tf = min(te, tf), max(te, tf)
+    tg = torch.rand_like(ta)
+    th = torch.rand_like(ta)
+    tg, th = min(tg, th), max(tg, th)
+
+    w = bm(ta, tb)
+    w2 = bm(tc, td)
+
+    bm.to(torch.float32)
+    w_ = bm(ta, tb)
+    w2_ = bm(tc, td)
+    w3_ = bm(te, tf)
+    w4_ = bm(tg, th)
+
+    bm.to(torch.float64)
+    w3 = bm(te, tf)
+    w4 = bm(tg, th)
+
+    assert w.dtype == w2.dtype == w3.dtype == w4.dtype == torch.float64
+    assert w_.dtype == w2_.dtype == w3_.dtype == w4_.dtype == torch.float32
+    assert torch.allclose(w, w_.double())
+    assert torch.allclose(w2, w2_.double())
+    assert torch.allclose(w3, w3_.double())
+    assert torch.allclose(w4, w4_.double())
+
+
+@pytest.mark.skip("BrownianInterval.to not supported")
```

Comment:
Why is this test skipped by default?

Reply:
Because of the issues with `BrownianInterval.to` as below.

Reply:
Same here.